### PR TITLE
[DOCFIX] Update deploy dora on k8s doc

### PR DESCRIPTION
**Pull request by ssz1997**
As we have (about to have) a proper helm chart for dora, update the doc to a more detailed page.

### CODE REVIEW

**Review by JiamingMai — APPROVED** [review]
LGTM

File: docs/en/dora/Deploy-Alluxio-Kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+### Step 3
+
+Prepare a configuration file `config.yaml`.
+All configurable properties can be found in file `values.yaml` from the code downloaded in step 1.
+
+The following properties must be included to enable Dora.
+```yaml
+properties:
+  alluxio.dora.client.ufs.root: <under_fs_uri>
```
Comment (huanghua78) [inline docs/en/dora/Deploy-Alluxio-Kubernetes.md:72]: I think these property keys are required too.
```
alluxio.dora.client.read.location.policy.enabled=true
alluxio.master.worker.register.lease.enabled=false
```

Comment (ssz1997) [reply docs/en/dora/Deploy-Alluxio-Kubernetes.md:72]: Helm chart will automatically add these two properties when constructing `alluxio-site.properties`, and disable short circuit as well.

**Comment by beinan** [comment]
alluxio-bot, merge this please.

**Pull request merged**